terraform-sentinel/step1.md

```diff
@@ -0,0 +1,90 @@
+In this scenario, you will apply the Sentinel logic principles to a Terraform specific deployment.
```

Comment:
We should give users a brief overview of the actual process here. Something like "In this scenario you will make sure an S3 bucket is not public..." give them the mental model for what all the steps they are performing will mean.